### DISCUSSION TITLE
TP 4-Ejercicio

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#127] [Preguntas y respuestas] [answered]
TP 4 - Ejercicio 1

Hola, tenemos esta duda con @lautarotummino . ¿Con el doble puntero estamos accediendo a la dirección de memoria del puntero entero?

    int numero_entero = 10;
    int *puntero_entero = &numero_entero;
    int **puntero_de_puntero = &puntero_entero;
    printf("El valor es: %d\n", numero_entero);
    printf("La dirreccion de memoria: %p\n", puntero_entero);
    printf("La dirreccion de memoria del puntero?: %p\n", puntero_de_puntero);

    size_t tamanio_entero = sizeof(int);
    printf("El tamanio entero es de : %lu byte\n", tamanio_entero);

**Comment by martinvilu** [comment]
Si, termina siendo un puntero doble

**Comment by JeanFranco99** [comment, marked as answer]
Un puntero a un puntero a entero `int **` apunta a datos `int *`, que a su vez apuntan a datos `int`.
